react/components/atoms/lightbox/lightbox.js

```diff
@@ -39,8 +54,36 @@ export class Lightbox extends PureComponent {
 
     constructor(props) {
         super(props);
+
+        this.screenWidth = Dimensions.get("window").width;
+        this.screenHeight = Dimensions.get("window").height;
+
+        this.fullscreenContainerBackgroundColor = 0;
+        this.fullscreenContainerBackgroundColorAnimated = 1;
+        this.fullscreenContainerTranslateYAnimationValue = 15;
+        this.fullscreenContainerBackgroundColorAnimatedDuration = 200;
+
+        this.fullZoomedInValue = 2;
+        this.fullZoomedOutValue = 1;
+
+        this.translatedX = 0;
+        this.translatedXTreshold = 0;
+        this.translatedY = 0;
+        this.translatedYTreshold = 0;
+
+        this.baseScale = new Animated.Value(1);
+        this.scaleRate = new Animated.Value(1);
+        this.lastScale = 1;
+
         this.state = {
-            visible: false
+            imageSizes: null,
+            zoomed: false,
+            zooming: false,
+            visible: this.props.visible,
+            fullscreenContainerBackgroundColor: new Animated.Value(0),

```

Comment:
perhaps only `containerBackgroundColor` or `backgroundColor` since there is no other background color being set